jax/_src/pallas/pallas_call.py

```diff
@@ -1009,6 +1009,41 @@ def pallas_call(
     name: str | None = None,
     compiler_params: dict[str, Any] | None = None,
 ) -> Callable[..., Any]:
+  """Invokes a Pallas kernel on some inputs.
+
+  See `Pallas Quickstart <https://jax.readthedocs.io/en/latest/pallas/quickstart.html>`_.
+
+  Args:
+    f: the kernel function.
+    out_shape: a PyTree of :class:`jax.ShapeDtypeStruct` describing the shape
+      and dtypes of the outputs.
+    grid_spec: TO BE DOCUMENTED.
+    debug: if True, Pallas prints various intermediate forms of the kernel
+      as it is being processed.
+    grid: the iteration space, as a tuple of integers. The kernel is executed
+      as many times as ``prod(grid)``. The default value ``None`` is equivalent
+      to ``()``.
+      See details at :ref:`pallas_grid`.
+    in_specs: a PyTree of :class:`jax.experimental.pallas.BlockSpec` with
+      a structure matching that of the positional arguments.
+      See details at :ref:`pallas_blockspec`.
+    out_specs: a PyTree of :class:`jax.experimental.pallas.BlockSpec` with
+      a structure matching that of the outputs.
+      See details at :ref:`pallas_blockspec`.
+      The default value for `out_specs` specifies the whole array,
+      e.g., as `pl.BlockSpec(lambda *indices: indices, x.shape)`.
+    input_output_aliases: TO BE DOCUMENTED.
+    interpret: runs the ``pallas_call`` as a ``jax.jit`` of a scan over the
+      grid whose body is the kernel lowered as a JAX function. This does not
+      require a TPU or a GPU, and it is useful for debugging.
+    name: TO BE DOCUMENTED.
+    compiler_params: TO BE DOCUMENTED.
+
+  Returns:
```

Comment:
Should we say that the function receiver both input and output refs here or in the documentation of `f` above?

Reply:
Done